doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-permissions-con.adoc

```diff
@@ -0,0 +1,12 @@
+[id='business-central-security-management-permissions-con']
+
+= Permissions
+
+A permission is an authorization granted to the user to do some actions related to a specific resource within the application. For example, user can have following permissions:
```

Comment:
change "to do some actions" to "perform actions"

Reply:
Done